lib/core/ElementRegistry.js

```diff
@@ -137,6 +137,29 @@ ElementRegistry.prototype.filter = function(fn) {
   return filtered;
 };
 
+/**
+ * Return the first elements that satisfies the provided testing function.
+ *
+ * @param {Function} fn
+ *
+ * @return {djs.model.Base}
+ */
+ElementRegistry.prototype.find = function(fn) {
+
+  var map = this._elements;
+
+  for (var i = 0; i < Object.keys(map).length; i++) {
```

Comment:
what about storing the keys in a separate variable, so you don't have to execute it twice?

```js
var keys = Object.keys(map)
```

Reply:
Agree, I will do that.